### ISSUE TITLE
train, stop, and continue training of a SequenceTagger, score 0.0

### ISSUE THREAD

**Issue by antgr** [flairNLP/flair#870]
Following tutorial 7, I run the following code:
```
from flair.data import Corpus
from flair.datasets import WNUT_17
from flair.embeddings import TokenEmbeddings, WordEmbeddings, StackedEmbeddings
from typing import List

# 1. get the corpus
corpus: Corpus = WNUT_17().downsample(0.1)

# 2. what tag do we want to predict?
tag_type = 'ner'

# 3. make the tag dictionary from the corpus
tag_dictionary = corpus.make_tag_dictionary(tag_type=tag_type)

# 4. initialize embeddings
embedding_types: List[TokenEmbeddings] = [
    WordEmbeddings('glove')
]

embeddings: StackedEmbeddings = StackedEmbeddings(embeddings=embedding_types)

# 5. initialize sequence tagger
from flair.models import SequenceTagger

tagger: SequenceTagger = SequenceTagger(hidden_size=256,
                                        embeddings=embeddings,
                                        tag_dictionary=tag_dictionary,
                                        tag_type=tag_type,
                                        use_crf=True)

# 6. initialize trainer
from flair.trainers import ModelTrainer
from flair.training_utils import EvaluationMetric

trainer: ModelTrainer = ModelTrainer(tagger, corpus)

# 7. start training
trainer.train('resources/taggers/example-ner',
              EvaluationMetric.MICRO_F1_SCORE,
              learning_rate=0.1,
              mini_batch_size=32,
              max_epochs=150,
              checkpoint=True)
```

But I have for each epoch, a similar output to the following, that says I have zero score:
`
2019-07-09 19:49:02,893 ----------------------------------------------------------------------------------------------------
2019-07-09 19:49:02,897 EPOCH 12 done: loss 5.0782 - lr 0.0250 - bad epochs 2
2019-07-09 19:49:04,156 DEV : loss 7.561308860778809 - score 0.0
2019-07-09 19:49:05,700 TEST : loss 6.32382345199585 - score 0.0
2019-07-09 19:49:12,848 ----------------------------------------------------------------------------------------------------
and finaly:
MICRO_AVG: acc 0.0 - f1-score 0.0
MACRO_AVG: acc 0.0 - f1-score 0.0
corporation tp: 0 - fp: 0 - fn: 6 - tn: 0 - precision: 0.0000 - recall: 0.0000 - accuracy: 0.0000 - f1-score: 0.0000
creative-work tp: 0 - fp: 0 - fn: 18 - tn: 0 - precision: 0.0000 - recall: 0.0000 - accuracy: 0.0000 - f1-score: 0.0000
group      tp: 0 - fp: 0 - fn: 15 - tn: 0 - precision: 0.0000 - recall: 0.0000 - accuracy: 0.0000 - f1-score: 0.0000
location   tp: 0 - fp: 0 - fn: 12 - tn: 0 - precision: 0.0000 - recall: 0.0000 - accuracy: 0.0000 - f1-score: 0.0000
person     tp: 0 - fp: 0 - fn: 38 - tn: 0 - precision: 0.0000 - recall: 0.0000 - accuracy: 0.0000 - f1-score: 0.0000
product    tp: 0 - fp: 0 - fn: 15 - tn: 0 - precision: 0.0000 - recall: 0.0000 - accuracy: 0.0000 - f1-score: 0.0000
`
Am I doing something wrong?

**Comment by songtaoshi** [comment]
hello,@antgr do find the error?

**Comment by antgr** [comment]
Hi @songtaoshi , No. I will update the issue if I find anything.

**Comment by marcelgwerder** [comment]
@antgr Ever found the cause for the zero score issue? I'm seeing similar output...

**Comment by antgr** [comment]
Hi guys, I have abandoned this issue because it was only experimentation. I currently have not time to go again to this. @alanakbik any idea on this?
